### DISCUSSION TITLE
umi4和umi max区别？

### DISCUSSION THREAD

**Discussion by joraboy328** [umijs/umi#12058] [General]
请教大家umi4和umi max啥关系？相互有啥区别？谢谢！

**Comment by fz6m** [comment]
umi 4 就是一个不包含任何插件的纯净版 umi ，max 就是 umi 4 + 一堆内置插件的包装版，如果你要用某个 umi 官方提供的插件功能，如果用 umi 就需要自己安装然后配置这个插件才能用，如果用 max 就是已经内置了直接开启就能用，但用 max 的话你也可以选择性开启你想要哪些功能，区别就是 max 的依赖安装的会比较多，综合来说还是推荐用 max ，因为如果你要用到官方提供的插件功能（如 qiankun 等），就不用自己再额外安装了。

FYI：

 - https://umijs.org/docs/guides/use-plugins#%E4%BD%BF%E7%94%A8%E6%8F%92%E4%BB%B6
